test/test_autograd.py

```diff
@@ -3773,10 +3773,16 @@ def backward(ctx, go):
         MyFunction.apply(inp).sum().backward(create_graph=True)
 
     def test_power_function(self):
-        a = torch.tensor([0., 0., 0.])
-        b = torch.tensor([-1., 0., 1.], requires_grad=True)
+        a = torch.tensor([0., 0., 0., 1.0])
+        b = torch.tensor([-1., 0., 1., 1.0], requires_grad=True)
         c = torch.sum(a**b)
         c.backward()
+        self.assertEqual(b.grad, torch.tensor([-inf, 0., 0., 0.0]), allow_inf=True)
```

Comment:
nit: if you keep this `0.` to be consistent with the other ones.

Reply:
Done